CONTRIBUTORS.md

```diff
@@ -0,0 +1,26 @@
+# Contributors
+
+This project is the result of the dedicated efforts of many people. It is because of their generous contributions that open-source project like this one are alive.
+
+This is the place to recognize their work.
+___
+
+- **Paul Götze**: Creator of *adoptoposs*.
+	- [GitHub](https://github.com/paulgoetze)
+	- [Twitter](https://twitter.com/paulgoetze)
+- **Enitan Williams**: Contributor.
+	- [GitHub](https://github.com/akhanaton)
+- **Florian Kraft**: Contributor.
```

Comment:
Would you like to be mentioned as co-maintainer, @floriank? Might also make it more clear who can be addressed regarding any concerns etc.